### PR TITLE
assets management rewrite

### PR DESCRIPTION
**Pull request by DavidBadura**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "name": "talkly",
+  "scripts": {
+    "postinstall": "./node_modules/.bin/bower install && ./node_modules/.bin/gulp",
+    "start": "./node_modules/.bin/gulp watch"
+  },
+  "dependencies": {
+    "bower": "^1.7.9",
+    "gulp": "^3.9.0",
+    "gulp-concat": "^2.6.0",
+    "gulp-minify-css": "^1.2.1",
```
Comment (smoench) [inline package.json]: @DavidBadura this gulp plugin is deprecated! But why don't you use the `outputStyle: compressed`-Option https://github.com/sass/node-sass#outputstyle of gulp-sass/node-sass which is written in c/c++?

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    sass = require('gulp-sass'),
+    concat = require('gulp-concat'),
+    minifyCSS = require('gulp-minify-css'),
+    browserSync = require('browser-sync').create()
+    ;
+
+gulp.task('default', ['css', 'js', 'fonts']);
+
+gulp.task('css', function () {
+    gulp.src(['src/scss/app.scss'])
```
Comment (smoench) [inline gulpfile.js]: @DavidBadura this gulp stream should be returned, so gulp is knowing then this task is finished

Comment (DavidBadura) [inline gulpfile.js]: Nice tip, I will change this

File: package.json (hunk 1 above)
Comment (DavidBadura) [inline package.json]: In this case, yes we have only sass here. But if you combine it with vendor css files then you need to compress them in diferent way. But yes, I fix this.

Btw. We are thinking about switching to reactjs with webpack.

**Comment by DavidBadura** [comment]
@tolry ready

File: install.sh [hunk 3]
```diff
@@ -8,13 +8,17 @@ cd `dirname $0`
 echo "installing composer dependencies (php)"
 composer install
 
+echo "use the right node version (nvm)"
+nvm use
```
Comment (tolry) [inline install.sh]: no actual version number required?

Comment (tolry) [inline install.sh]: nvm - didn't see the .nvrmc file

**Comment by tolry** [comment]
@DavidBadura thx

**Pull request merged**